lib/check-dependencies.js

```diff
@@ -202,7 +202,36 @@ const checkDependenciesHelper = (syncOrAsync, config, callback) => {
             return;
         }
 
-        const depVersion = require(depJson).version;
+        const depJson = require(depJsonPath);
+
+        // Support NPM package aliases
+        if (
+            options.packageManager === 'npm' &&
+            depJson &&
+            depJson._requested &&
+            depJson._requested.type === 'alias'
```

Comment:
This is internal npm metadata that doesn't exist in Yarn. Is that necessary? Can't we just detect the alias syntax by parsing the value as done below?